unison-src/transcripts/merges.output.md

```diff
@@ -112,13 +112,13 @@ We can also delete the fork if we're done with it. (Don't worry, it's still in t
   Note: The most recent namespace hash is immediately below this
         message.
   
-  ⊙ #4dmogr46d2
+  ⊙ #218mq9e558
```

Comment:
I believe these changed hashes are due to removing redundant steps on child branches in stepManyAt.